grammar.y

```diff
@@ -4,13 +4,33 @@ package sqlparser
 import "bytes"
 
 var keywordsNotAllowed = map[string]struct{}{
-  // non-deterministic keywords
+  // We don't allow non-deterministic keywords as identifiers. 
 	"CURRENT_TIME":      {},
 	"CURRENT_DATE":      {},
 	"CURRENT_TIMESTAMP": {},
 
-  // SQLite reserved keywords
+  // SQLite reserved keywords that are not part of Tableland spec.
+  // We can't allow them as identifiers because it will throw an error in SQLite.
+  //
+  // SQLite has more reserved keywords (eg. CREATE, INSERT, ...). But those are part of the Tableland grammar,
+  // that means that the parser already checks from them.
+  //
+  // These were identified by running the `TestReservedKeywords` test.
   "REFERENCES" : {},
```

Comment:
Adding more keywords to the list. If these keywords are used as identifiers they would cause the same problem that `REFERENCES` caused. 

These keywords were identified by running a code ([TestReservedKeywords](https://github.com/tablelandnetwork/sqlparser/pull/9/files#diff-eff279f70f1a81e2abdae4e57736a72120ce3a184bc25ccdbbb60cb10b3027c5R4609)) that differentiates which keywords SQLite allows to be used as an identifier and which it does not. From the list of all reserved keywords, we have to exclude those that are already part of Tableland spec. The result is the below list. 

To double-check:
- Run [TestReservedKeywords](https://github.com/tablelandnetwork/sqlparser/pull/9/files#diff-eff279f70f1a81e2abdae4e57736a72120ce3a184bc25ccdbbb60cb10b3027c5R4609) with `-v`
- Get the reserved keywords from the result above and compare with [keywords](https://github.com/tablelandnetwork/sqlparser/blob/2b9dbbbf4507afaa11acaa511a23765a4c79a691/lexer.go#L10)
- Exclude the reserved words that are already part of the Tableland spec
- The remaining should be ones below

Reply:
Good exhaustive scan. :+1: